doc/api/globals.md

```diff
@@ -1199,6 +1199,20 @@ added: v18.0.0
 
 A browser-compatible implementation of [`WritableStreamDefaultWriter`][].
 
+## Web Storage API
+
+In Node.js, the localStorage and sessionStorage objects function differently compared to browsers or Deno:
+
+* Both `localStorage` and `sessionStorage` are scoped to the current process,
+  not individual users or server requests. This is crucial for applications
+  like server-side rendering.
+* `localStorage` uses the value of the `--localstorage-file` flag as its
+  origin. This file can be accessed simultaneously by multiple processes,
+  which may require implementing file locking or other synchronization
+  mechanisms to ensure data integrity.
+* The storage quota for both `localStorage` and `sessionStorage` is
+  10MB per process.
```

Comment:
I think this information is already present in this file under `localStorage` and `sessionStorage`